nextjs-app/package.json

```diff
@@ -16,18 +16,20 @@
     "@justeattakeaway/pie-css": "0.8.0",
     "@justeattakeaway/pie-divider": "0.9.2",
     "@justeattakeaway/pie-icon-button": "0.22.0",
-    "@justeattakeaway/pie-icons-webc": "0.11.1",
+    "@justeattakeaway/pie-icons-webc": "0.15.0",
     "@justeattakeaway/pie-link": "0.11.2",
     "@justeattakeaway/pie-modal": "0.34.0",
     "@justeattakeaway/pie-spinner": "0.3.0",
     "@justeattakeaway/pie-switch": "0.18.0",
     "@lit-labs/nextjs": "0.1.4",
     "@lit/react": "1.0.2",
+    "add": "^2.0.6",
     "lit": "3.1.0",
     "next": "13.5.6",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "sass": "1.69.5"
+    "sass": "1.69.5",
+    "yarn": "^1.22.21"
```

Comment:
yarn probably shouldn't be added as dev dep. Was this done for any reason in particular? I notice we're using Yarn 3 at the root. - https://github.com/justeattakeaway/pie-aperture/tree/main/.yarn/releases

Reply:
I have absolutely no idea how that got there - will remove now! Nice spot